generic_starter.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+
+def parse_input(filename: str) -> list:
+    """
+    Reads input from file
+    T: num of test cases to follow
+
+    TASK: 
+    returns: list[]
+    """
+    with open(filename) as f:
+        num_test_cases = int(f.readline())
+        test_cases = []
+        for _ in range(num_test_cases):
+            test_cases.append(f.readline())
+    return test_cases
+
+
+def write_output(filename: str, values: list) -> None:
+    """
+    Writes the data to file in correct format for KickStart 2020 round B
```

Comment:
This block comment is referencing a specific challenge – could you generalize it?

I think a few of the other PRs also need this comment changing, if you could double check those too.

Reply:
Comment has been updated